### PR TITLE
Feature status bar candidate

### PR DESCRIPTION
**Pull request by roman-se**
Separate status bar component added

### CODE REVIEW

File: src/components/Candidate.vue [hunk 1]
```diff
@@ -11,66 +11,85 @@
...
-                      i.ion-edit.m-r-10
-                      | Edit Details
+              .col-4.m-r-15
+                  status-bar.m-b-30(
+                    value = "3",
+                    max = "5"
+                    )
+              .col.m-t-30.m-l-15
+                    .row.m-0
+                        p.m-0(style={color: '#f4a625', fontWeight: 'bold', fontSize: '2em'})
```
Comment (jen-huang) [inline src/components/Candidate.vue]: move inline styles out

File: src/main.js [hunk 2]
```diff
@@ -36,6 +36,9 @@ import PrepQuestion from './components/PrepQuestion.vue'
 import PrepOnboarding from './components/PrepOnboarding.vue'
 import PrepDashboard from './components/PrepDashboard.vue'
 
+//Plugins
```
Comment (jen-huang) [inline src/main.js]: change to `// UI components`

File: src/main.js [hunk 3]
```diff
@@ -67,6 +70,8 @@ if(document.getElementById('app-public')) {
 
 // App routes
 if(document.getElementById('app')) {
+    //adding statuc bar plugin component
+    Vue.component('status-bar', StatusBar)
```
Comment (jen-huang) [inline src/main.js]: remove comment
add line break after this

attach it to the app-prep section as well

**Comment by roman-se** [comment]
The issues mentioned in the review have been fixed

File: src/components/Candidate.vue [hunk 4]
```diff
@@ -92,6 +92,14 @@
                 p Answer Preview
 </template>
 
+<style lang="scss">
```
Comment (jen-huang) [inline src/components/Candidate.vue]: add "scoped" attribute so that this style only affects this component - see PrepIzzy for example

**Pull request merged**